_docs/_latest/configure/inventory/gsuite.md

```diff
@@ -5,41 +5,50 @@ order: 201
 
 #  {{ page.title }}
 
-This page describes how to enable the data collection of G Suite for processing by Forseti Inventory.
+This page describes how to enable the data collection of G Suite for processing
+by Forseti Inventory.
 
 ---
 
-## Enable Domain-Wide Delegation in G Suite
+## Before you begin
 
-To enable collection of G Suite data, follow the steps below to enable your existing forseti
-service account for this functionality. Read more about 
-[domain-wide delegation](https://developers.google.com/identity/protocols/OAuth2ServiceAccount?hl=en_US#delegatingauthority).
+To complete this guide and enable a service account in your G Suite admin
+control panel, you must have the **super admin** role in admin.google.com.
+
+## Enable Domain-wide Delegation (DwD) in G Suite
```

Comment:
This might break a few links we'll need to search the code and the docs for any links to the previous heading:

`enable-domain-wide-delegation-in-g-suite`

Reply:
@joecheuk

Reply:
Can we do custom anchor names? I haven't been able to find a reference as to whether that's possible in GitHub. Something like {:#anchor_name} works in some markdown; if it would work here, we could just do {:#enable-domain-wide-delegation-in-g-suite}

Reply:
I'm not sure on custom anchor links, but I believe this works [doc]({% link _some/path.md}#anchor-tag)

Reply:
Resolved.